.scripts/electron-desktop-environment/desktop-environment-content-factory.ts

```
@@ -32,6 +33,11 @@ export class DesktopEnvironmentContentFactory {
 				return common
 					.generate(environment)
 					.concat(apiServer.generate(environment));
+			case 'gauzy-agent':
+				const gauzyAgent = new GauzyAgentEnvironmentContent();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)